include/PianoRoll.h

```diff
@@ -253,6 +255,13 @@ protected slots:
 		PR_BLACK_KEY
 	};
 
+	enum GridMode
+	{
+		gridNudge,
+		gridSnap,
+		gridFree
```

Comment:
```suggestion
		//gridFree
```
I'd remove or comment this out while it's not implemented